### PR TITLE
TASK: README improvements

### PR DESCRIPTION
**Pull request by crydotsnake**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,61 +62,88 @@ Please note that we use Python 3 only, so make sure that you use correct version
...
 
 Create a new virtual environment (python 3.9) if needed. Then, install all the required dependencies.
 The dependencies are compiled by [pip-tools](https://github.com/jazzband/pip-tools), which
 compiles `requirements.txt` ensuring compatibility between packages.
 
-    pip install pip-tools
-    pip-sync
+```bash
+pip install pip-tools
+```
```
Comment (marksweb) [inline README.md:83]: I'd put this in 1 block as there's nothing between @crydotsnake 

```bash
pip install pip-tools
pip-sync
```

File: README.md [hunk 2]
```diff
@@ -228,10 +266,13 @@ from `requirements.txt`.
 
 For example:
 
-    pip-compile -U
-
-    pip-sync
+```bash
+pip-compile -U
+```
```
Comment (marksweb) [inline README.md:271]: Again I'd do this as 1 block @crydotsnake 

```bash
pip-compile -U
pip-sync
```

**Review by marksweb — CHANGES_REQUESTED** [review]

**Review by amakarudze — COMMENTED** [review]
Thanks @crydotsnake for your work on this. If you can attend to the changes requested by @marksweb, I will be happy to merge this in.

**Comment by crydotsnake** [comment]
Done! :) @amakarudze

**Review by marksweb — APPROVED** [review]

**Review by amakarudze — APPROVED** [review]

**Comment by amakarudze** [comment]
> Done! :) @amakarudze

Thanks @crydotsnake 👍 . Merging this now!

**Pull request merged**